chat/internal/delivery/grpc/list_user_chats.go

```diff
@@ -0,0 +1,45 @@
+package delivery_grpc
+
+import (
+	"context"
+	chat "msa_big_tech/chat/pkg/proto/v1"
+)
+
+func (s *Implementation) ListUserChats(ctx context.Context, req *chat.ListUserChatsRequest) (*chat.ListUserChatsResponse, error) {
+
+	res, err := s.srv.GetListUserChats(ctx, req.UserId)
+	if err != nil {
+		return nil, err
+	}
+
+	coverArr := make([]*chat.Chat, len(res))
```

Comment:
С такой инициализаций ниже баг.

Возможно хотели так?  

```suggestion
	coverArr := make([]*chat.Chat, 0, len(res))
```